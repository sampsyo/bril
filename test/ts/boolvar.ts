let x: boolean = true;
if (x) {
  console.log(1);
} else {
  console.log(2);
}
