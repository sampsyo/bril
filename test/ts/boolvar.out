main {
  v0: bool = const true;
  x: bool = id v0;
  v2: bool = id x;
  br v2 then.1 else.1;
then.1:
  v3: int = const 1;
  print v3;
  v4: int = const 0;
  jmp endif.1;
else.1:
  v5: int = const 2;
  print v5;
  v6: int = const 0;
endif.1:
}
